### PR TITLE
Payment gui

### PR DESCRIPTION
**Pull request by badb**

### CODE REVIEW

**Comment by chfast** [comment]
## [Current coverage](https://codecov.io/github/imapp-pl/golem?ref=0e145efac960c4a8c8bec41a0e258d2d553ac251) is `28.31%`

> Merging **#112** into **develop** will increase coverage by **+0.20%** as of [`0e145ef`](https://codecov.io/github/imapp-pl/golem/commit/0e145efac960c4a8c8bec41a0e258d2d553ac251)

``` diff
@@            develop    #112   diff @@
=======================================
  Files           252     255     +3
  Stmts         19982   20147   +165
  Branches       2270    2282    +12
  Methods           0       0       
=======================================
+ Hit            5617    5704    +87
  Partial         220     220       
- Missed        14145   14223    +78
```

> Review entire [Coverage Diff](https://codecov.io/github/imapp-pl/golem/compare/8050819f532ffe7cc632e03f93c32be1a1b8ec10...0e145efac960c4a8c8bec41a0e258d2d553ac251) as of [`0e145ef`](https://codecov.io/github/imapp-pl/golem/commit/0e145efac960c4a8c8bec41a0e258d2d553ac251)

---

### [Uncovered Suggestions](https://codecov.io/github/imapp-pl/golem/features/suggestions?ref=0e145efac960c4a8c8bec41a0e258d2d553ac251)
1. `+0.14%` via [.../task/pbrtgnrtask.py#70...97](https://codecov.io/github/imapp-pl/golem/gnr/task/pbrtgnrtask.py?ref=0e145efac960c4a8c8bec41a0e258d2d553ac251#70) 
2. `+0.09%` via [gnr/task/vraytask.py#278...295](https://codecov.io/github/imapp-pl/golem/gnr/task/vraytask.py?ref=0e145efac960c4a8c8bec41a0e258d2d553ac251#278) 
3. `+0.09%` via [.../task/pbrtgnrtask.py#1...18](https://codecov.io/github/imapp-pl/golem/gnr/task/pbrtgnrtask.py?ref=0e145efac960c4a8c8bec41a0e258d2d553ac251#1) 
4. _[See 7 more...](https://codecov.io/github/imapp-pl/golem/features/suggestions?ref=0e145efac960c4a8c8bec41a0e258d2d553ac251)_

> Powered by [Codecov](https://codecov.io). Updated on successful CI builds.

**Comment by badb** [comment]
Should be developed further in the future, when we define all requirements.

File: golem/task/taskserver.py [hunk 1]
```diff
@@ -543,11 +545,13 @@ def __connection_for_result_rejected_failure(self, conn_id, key_id, subtask_id):
             pc.time = time.time()
 
     def __connection_for_pay_for_task_established(self, session, conn_id, key_id, task_id, price):
+        print "CONNECTION FOR PAY FOR TASK ESTABLISHED"
```
Comment (chfast) [inline golem/task/taskserver.py:548]: Replace with logs?

**Comment by chfast** [comment]
This looks ok. Can we see screenshots before merging?

**Comment by badb** [comment]
It's just something like that right now: https://drive.google.com/file/d/0B8jXV0W-_NcWell0dHljTzFfTkE/view?usp=sharing You can sort columns. My vision: There will be a button in transaction field (or link) that will allow you to see proper transaction in blockchain. There should be also added column with date. 
Income tab: https://drive.google.com/file/d/0B8jXV0W-_NcWcDJPSl9id0ttcVU/view?usp=sharing

**Comment by chfast** [comment]
:+1:

**Pull request merged**